### ISSUE TITLE
Cannot configure backlght on Polaris 15 AMD Gen3

### ISSUE THREAD

**Issue by chrosey** [tuxedocomputers/tuxedo-keyboard#103]
I tried installing package over various methods
- apt
- make
- make dkms

Changing values in /etc/modprobe.d/tuxedo_keyboard.conf have no effect. 
modprobe ... <parameter> has no effect
echo ... > sudo tee /sys/... no permission

Setting values like RED in color_left produces log entries on reboot because value is not supported. so it seems to work basically. but  I have no change to configure anything.

What can I do?

**Comment by StevenSeifried** [comment]
Are you running Ubuntu? You must install [tuxedo-keyboard-ite](http://deb.tuxedocomputers.com/ubuntu/pool/main/t/tuxedo-keyboard-ite/).

**Comment by chrosey** [comment]
if that does also matches for tuxedo-os? i'll try after work

**Comment by chrosey** [comment]
tuxedo-keyboard-ite was already installed. also reinstalling did not help

**Comment by tuxedoxt** [comment]
Hello,

`tuxedo-keyboard-ite` is actually not needed for Polaris.

Try setting the `color` parameter of `tuxedo-keyboard` to `RED` instead.

There's a provisional interface for on the fly changes in ` /sys/devices/platform/tuxedo_keyboard/`.

Here's a link for reference (see option 2 for polaris): https://www.tuxedocomputers.com/en/Infos/Help-Support/Instructions/Installation-of-keyboard-drivers-for-TUXEDO-Computers-models-with-RGB-keyboard-.tuxedo

**Comment by chrosey** [comment]
this seems not correct. if i uninstall `tuxedo-keyboard-ite`, keyboard backlight is turned off.

i tried the other steps mentioned in the url, but i don't even have the path `/sys/devices/platform/tuxedo_keyboard/uw_kbd_bl_color/`

**Comment by tuxedoxt** [comment]
Ok, let's check the model

```
cat /sys/class/dmi/id/board_name
```

**Comment by chrosey** [comment]
sure
```
cat /sys/class/dmi/id/board_name
GMxZGxx
```

**Comment by tuxedoxt** [comment]
Ah, this should be a *Stellaris AMD gen 3*, which would explain why it does need `tuxedo-keyboard-ite`.

Although the keyboard backlight for this model theoretically has more fancy features, the driver only has a very basic interface so far. This will be extended in the future.

**Comment by chrosey** [comment]
oh you're right...
sorry, i've read to much about polaris on keyboard-backlighting. 

so there is no misconfiguration but it just does not work yet?

**Comment by tuxedoxt** [comment]
Correct, driver for this keyboard so far supports the basics, white "color" with brightness control.

**Comment by DeRaafMedia** [comment]
I'm confused I have a Polaris 15 Gen3, This is my keyboard model

```
cat /sys/class/dmi/id/board_name
GMxZGxx
```

After reading this thread I believe I can NOT change the backlight colors (made a few attempts which failed)? Any time line when this will be possible. Would love to change the color in certain situations.
